BDT_application/step2_BDTapplication_condor/wrapper.sh

```diff
@@ -11,47 +11,7 @@ HOME=$PWD
 # copy BDT weights, one central weight and six systematic weights
 mkdir BDT_weights_0
 cd BDT_weights_0
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_0/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_central/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_1
-cd BDT_weights_1
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_1/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_jesup/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_2
-cd BDT_weights_2
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_2/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_jesdo/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_3
-cd BDT_weights_3
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_3/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_jerup/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_4
-cd BDT_weights_4
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_4/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_jerdo/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_5
-cd BDT_weights_5
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_5/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_metUnslusEnup/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_6
-cd BDT_weights_6
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_6/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_metUnslusEndo/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_7
-cd BDT_weights_7
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_7/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_muPtup/weights/TMVAClassification_BDTG.weights.xml .
-cd $HOME
-
-mkdir BDT_weights_8
-cd BDT_weights_8
-cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_weights/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_8/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_muPtdo/weights/TMVAClassification_BDTG.weights.xml .
+cp /eos/user/t/tihsu/database/ExtraYukawa_BDT/BDT_weight/YEAR/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS/ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_0/dataset_ttc_SIGTYPE_COUPLING_MSIGTYPEMASS_central/weights/TMVAClassification_BDTG.weights.xml .
```

Comment:
same comment as above , you just use the exisiting training files.
not need to point to your new training files.